src/components/LobLabel/LobLabel.vue

```diff
@@ -3,32 +3,34 @@
     <label
       :for="labelFor"
       :class="[
-        {'flex items-center justify-between mb-2 text-sm text-gray-500': !srOnlyLabel},
-        { 'sr-only': srOnlyLabel }
+        readOnly ? 'text-gray-300' : 'text-gray-800',
+        srOnlyLabel ? 'sr-only' : 'flex items-center mb-1 type-small-500'
       ]"
     >
-      <span>
-        {{ label }}
-        <span
-          v-if="required"
-          class="text-sm text-error"
-        >
-          *
-        </span>
-      </span>
       <Tooltip
         v-if="tooltipContent"
-        position="left"
+        position="right"
```

Comment:
wondering about the position of the tooltip, designs show two variations one on the right of the label and one on the left end?

Reply:
it also looks like a lot of space between tooltip and label, visually I mean - is the margin between then defined in the design?

Reply:
the design says 8px

Reply:
added